x-pack/plugins/observability/public/pages/alerts/containers/alerts_table_t_grid/alerts_table_t_grid.tsx

```diff
@@ -311,7 +310,7 @@ export function AlertsTableTGrid(props: AlertsTableTGridProps) {
 
   const [flyoutAlert, setFlyoutAlert] = useState<TopAlert | undefined>(undefined);
   const [tGridState, setTGridState] = useState<Partial<TGridModel> | null>(
-    JSON.parse(localStorage.getItem(ALERT_TABLE_STATE_STORAGE_KEY) ?? 'null')
+    JSON.parse(localStorage.getItem(stateStorageKey) ?? 'null')
```

Comment:
Isn't unsafe to parse JSON without a try/catch block? I believe there is a helper/util in Kibana that handles that. `/src/plugins/kibana_utils/public/storage/storage.ts`

Reply:
Ok let me check it

Reply:
@fkanout I made a quick search in the whole project for imports of `kibana_utils/public/storage`, but I didn't find anything. So I was wondering how I should probably use `Storage` from this file. My guess is that it should be part of a kbn pacakge? Do you have any idea? I can simply try to import this file, but I didn't find any other imports of it in the whole project, that's why I am wondering if there is a proper way to do it.

Reply:
Ok I found something. Here's how I can import it `import { IStorage, Storage } from '@kbn/kibana-utils-plugin/public';`. My guess is that this will require a bit more refactoring. Should it be part of this PR?

Reply:
@fkanout Ok in the end I did the refactoring as part of this PR https://github.com/elastic/kibana/pull/130538/commits/d12eeeab59926985dc26e277d32476d74cb7a295